### PR TITLE
vglass: replace surfman RPC in xcpmd, retire libdmbus

### PR DESCRIPTION
**Pull request by eric-ch**
xcpmd interacted with surfman for brightness control, this can be retired. Replace with vglass RPCs (including the `input_daemon` IDL, which is not provided by input-server).

### CODE REVIEW

**Comment by eric-ch** [comment]
v2: Rebase on master. No change.

File: xcpmd/src/acpi-events.c [hunk 1]
```diff
@@ -41,12 +41,9 @@ static struct event acpi_event;
...
 void adjust_brightness(int increase, int force) {
-
-    if ( force || (pm_quirks & PM_QUIRK_SW_ASSIST_BCL) || (pm_quirks & PM_QUIRK_HP_HOTKEY_INPUT)) {
-        if (increase)
-            com_citrix_xenclient_surfman_increase_brightness_(xcdbus_conn, SURFMAN_SERVICE, SURFMAN_PATH);
-        else
-            com_citrix_xenclient_surfman_decrease_brightness_(xcdbus_conn, SURFMAN_SERVICE, SURFMAN_PATH);
+    (void) increase;
+    if (force || (pm_quirks & PM_QUIRK_SW_ASSIST_BCL) || (pm_quirks & PM_QUIRK_HP_HOTKEY_INPUT)) {
+        xcpmd_log(LOG_WARNING, "Brightness adjustment not implemented yet.");
```
Comment (jandryuk) [inline xcpmd/src/acpi-events.c]: vglass idl defines increase/decrease_brightness, but it's unclear to me if they are hooked up.

File: xcpmd/src/acpi-events.c [hunk 2]
```diff
@@ -40,14 +40,6 @@ static struct event acpi_event;
 
 static struct ev_wrapper ** acpi_event_table;
 
-void adjust_brightness(int increase, int force) {
-    (void) increase;
-    if (force || (pm_quirks & PM_QUIRK_SW_ASSIST_BCL) || (pm_quirks & PM_QUIRK_HP_HOTKEY_INPUT)) {
-        xcpmd_log(LOG_WARNING, "Brightness adjustment not implemented yet.");
```
Comment (jandryuk) [inline xcpmd/src/acpi-events.c]: ... and it's removed here anyway.

**Review by jandryuk — APPROVED** [review]
Seems generally fine.

File: xcpmd/src/acpi-events.c (hunk 2 above)
Comment (eric-ch) [reply xcpmd/src/acpi-events.c]: The DBus call for vglass is added back in `handle_bcl_event`.

**Review by jandryuk — APPROVED** [review]

**Comment by jandryuk** [comment]
Merging soon

**Pull request merged**